test/integration/misc_test.go

```diff
@@ -41,8 +41,8 @@ func TestAPIMeta(t *testing.T) {
 		t.Error("Get returned no git addresses")
 	}
 
-	if !*meta.VerifiablePasswordAuthentication {
-		t.Error("APIMeta VerifiablePasswordAuthentication is false")
+	if *meta.VerifiablePasswordAuthentication {
+		t.Error("APIMeta VerifiablePasswordAuthentication is true")
```

Comment:
What's the story with the complete reversal on this one?
Do we need to add a comment? Or was it just plain broken?

Reply:
`verifiable_password_authentication` is `false` for `github.com` since February 14, 2020.  So yeah, the test seems broken for 4.5 years.

I found this information on:

- https://developer.github.com/changes/2020-02-14-deprecating-oauth-auth-endpoint/
- https://github.com/desktop/desktop/issues/9231
- https://docs2.lfe.io/v3/meta/#:~:text=GitHub%20Enterprise%20instances%20using%20CAS%20or%20OAuth%20for%20authentication%20will%20return%20false

Reply:
@gmlewis I can remove this check at all:

```go
	if *meta.VerifiablePasswordAuthentication {
		t.Error("APIMeta VerifiablePasswordAuthentication is true")
	}
```

The test will still be valuable.